src/commands/copyIpAddress.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../extensionVariables';
+import { localize } from '../localize';
+import { VirtualMachineTreeItem } from '../tree/VirtualMachineTreeItem';
+
+export async function copyIpAddress(context: IActionContext, node?: VirtualMachineTreeItem): Promise<void> {
+    if (!node) {
+        node = await ext.tree.showTreeItemPicker<VirtualMachineTreeItem>(VirtualMachineTreeItem.contextValue, context);
+    }
+
+    await vscode.env.clipboard.writeText(await node.getHostName());
```

Comment:
So does this actually copy the ip address or the host name? Seems like the ipAddress...in which case can you rename `getHostName` to `getIpAddress`

Reply:
Ah, you're right.  Originally, this was used only configureSshConfig, in which case the IP address _was_ the host name for the config file, but it is generically the IP address.